src/components/tableOfContents/index.tsx

```diff
@@ -98,7 +98,7 @@ export function TableOfContents() {
         }
         return {
           depth: el.tagName === 'H2' ? 2 : 3,
-          url: `#${el.id} `,
+          url: `#${el.id}`,
```

Comment:
Firefox does not trim the `%20` from the link being generated here, so the `On This Page` table of contents was not working for any page.

Chrome seems to trim this, so it worked fine, but this will fix Firefox' behavior.

Reply:
Nice! Thanks for fixing this!